src/rpc/methods/state.rs

```diff
@@ -87,8 +87,6 @@ impl StateCall {
         let tipset = state_manager
             .chain_store()
             .load_required_tipset_or_heaviest(&tsk)?;
-        // Handle expensive fork error?
```

Comment:
Should we really remove this as it is relevant but it's not just planned yet.

Reply:
I'll let @elmattic weigh in here, given the TODO was assigned to him.

Reply:
I will flesh out the details of the issue. I think it's important enough to have one.